### PR TITLE
Add headers prop to InertiaLink components

### PR DESCRIPTION
**Pull request by pedroborges**
Declare new `headers` prop on the `InertiaLink` component for all adapters.

### CODE REVIEW

**Review by claudiodekker — APPROVED** [review]

**Comment by reinink** [comment]
Thanks @pedroborges! 👍

File: packages/inertia-vue/src/link.js [hunk 1]
```diff
@@ -30,6 +30,10 @@ export default {
     only: {
       type: Array,
       default: () => [],
+    },
+    headers: {
+      type: Object,
+      default: () => {},
```
Comment (reinink) [inline packages/inertia-vue/src/link.js:36]: Realize now that this is actually wrong. It needs to be:

```js
default: () => ({}),
```

Right now this results in a default of `undefined`.

I'll fix. 👍

Comment (pedroborges) [reply packages/inertia-vue/src/link.js:36]: Ops, sorry about that 🥺

Comment (reinink) [reply packages/inertia-vue/src/link.js:36]: Haha all good brother! I didn't catch it either. 😊

**Pull request merged**